public/static/locales/en/landing.json

```diff
@@ -1,11 +1,11 @@
 {
   "home": {
     "banner": {
-      "heading": "Find travel insurance providers if you have a serious medical condition",
+      "heading": "Find a travel insurance provider if</br> you have a serious medical condition",
```

Comment:
Although the request is to break the text after a specific word I think it would be better to do this at a particular width (as it is in the Welsh version). The text breaks awkwardly on smaller viewports this way.

Reply:
Not sure if this is possible because by breaking it on `providers` the second line is longer than the first and will break first

Reply:
OK, it's not a big deal from my point of view. I'm happy to approve the PR.